resources/database/populate_action_rules.sql

```diff
@@ -0,0 +1,9 @@
+/* Populate action rules for ready for live */
+
```

Comment:
Possibly rename this file to be more specific to a survey? rsi_populate_action_rules.sql?

Reply:
@tej99 Ah, are they specific to RSI? (It was copy-pasted from elsewhere by @madeye-matt - possibly prod).

Reply:
Looks like they are, the casetypedefault is specific to RSI (75b19ea0-69a4-4c58-8d7f-4458c8f43f5c) which is linked to the actionplan which is linked to the action rules.

Reply:
@tej99 I'll get it renamed.